Runtime/AvatarCreator/Scripts/Managers/AvatarManager.cs

```diff
@@ -76,6 +76,7 @@ public async Task<AvatarCreationResponse> CreateAvatarAsync(AvatarProperties ava
             }
             catch (Exception e)
             {
+                ThrowErrIfCancellationException(e);
```

Comment:
It's super annoying that we have to add this in all the exception parts of this script. 
It could be moved into some sort of "HandleException(ex)" function to reduce the code duplication.
It might also be nice in future to change the OnError action to have a parameter of type Exception. Then we can just use that for everthing (including throwing the cancellation exception). It would be a breaking change though.

Reply:
You can actually use the built in exclude logic to handle this, if you wanted a more graceful solution ->

`catch (Exception ex) when (ex is not OperationCanceledException)`

https://stackoverflow.com/questions/12633903/elegantly-handle-task-cancellation